### PR TITLE
fix(@desktop/onboarding): onboarding flows for `I already use Status` path do not match figma requirements

### PR DESCRIPTION
**Pull request by saledjenic**
Corresponding `status-go` PR:
- https://github.com/status-im/status-go/pull/2976

Fixes: #8467 

I recorded few short videos for fetching data from waku...

1. here you can see the fetching component behaviour in Storybook:

https://user-images.githubusercontent.com/86303051/204840562-863a8121-241f-4b3b-b19b-0185c08d777c.mov

2. here you can see Keycard login and fetching data from waku, when all backed up messages were fetched successfully before timeout:

https://user-images.githubusercontent.com/86303051/204840846-14868596-01b0-49a3-ba16-30ec9cce0390.mov

3. here you can see login using seed phrase, in case where profile data was not fetched before timeout (for this simulation timeout is set to 10 seconds):

https://user-images.githubusercontent.com/86303051/204840908-a7947834-9078-48f7-9ff1-6f3261838729.mov

### CODE REVIEW

File: ui/app/AppLayouts/Onboarding/views/ProfileFetchingView.qml [hunk 1]
```diff
@@ -59,56 +55,167 @@ Item {
...
+        State {
+            name: Constants.startupState.profileFetching
+            when: root.startupStore.currentStartupState.stateType === Constants.startupState.profileFetching
+            PropertyChanges {
+                target: title
+                text: qsTr("Fetching data...")
+            }
+            PropertyChanges {
+                target: button
+                text: d.getCurrTimerValue()
```
Comment (alexjba) [inline ui/app/AppLayouts/Onboarding/views/ProfileFetchingView.qml:189]: function getCurrTimerValue {..} could be reduced to

`text: Qt.formatTime(new Date(0,0,0,0,0,d.counter), "mm:ss")`

File: ui/app/AppLayouts/Onboarding/views/ProfileFetchingView.qml [hunk 2]
```diff
@@ -59,56 +55,167 @@ Item {
...
-                onClicked: {
-                    root.startupStore.doSecondaryAction()
+            Timer {
+                id: timer
+                interval: 1000
+                running: root.startupStore.currentStartupState.stateType === Constants.startupState.profileFetching
+                repeat: true
+                onTriggered: {
+                    d.counter--
+                    let timerVal = d.getCurrTimerValue()
```
Comment (alexjba) [inline ui/app/AppLayouts/Onboarding/views/ProfileFetchingView.qml:161]: Might be safer to check if d.counter == 0. This way the logic doesn't depend on the time format.

**Review by alexjba — COMMENTED** [review]
I've been reviewing mainly the UI part and LGTM. Added a few suggestions.

File: ui/app/AppLayouts/Onboarding/views/ProfileFetchingView.qml (hunk 2 above)
Comment (saledjenic) [reply ui/app/AppLayouts/Onboarding/views/ProfileFetchingView.qml]: @alexjba thanks, I overlooked that, updated.

**Review by osmaczko — APPROVED** [review]
This feature looks great!

Briefly reviewed the nim part, LGTM.

**Pull request merged**